spec/features/admin/merchants/index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "admin merchants index" do
+
+  describe "As an admin when I visit the admin merchants index (/admin/merchants)" do
+    it "I see the name of each merchant in the system" do
+      merchants = FactoryBot.create_list(:merchant, 10)
+
+      visit admin_merchants_path
+      
+      merchants.each do |merchant|
```

Comment:
That is a really interesting way to test. I never thought about using iteration like that.